applications/nrf_desktop/doc/settings_load.rst

```diff
@@ -0,0 +1,29 @@
+.. _settings_load:
+
+Settings load module
```

Comment:
I suggest to change the module name to `Setting loader`.

Reply:
This will be done after merging PR with changes in the code.